projects/sentinel/layers/cassandra.yml

```diff
@@ -1,22 +1,23 @@
 ---
-- name: smart-cassandra | Cluster setup
+- name: Setup Cassandra cluster
   hosts: localhost
   connection: local
   roles:
     - role: aws
       ec2_key_name: "eu-smartcat"
-      ec2_security_group: "anomaly-stack"
+      ec2_security_group: "sentinel"
```

Comment:
We should think of separating security groups (maybe add it as issues) since monitoring is using different stuff from Cassandra and it is different from SPark